### ISSUE TITLE
Plotting to pdf does not work wirth GUI

### ISSUE THREAD

**Issue by coquelin77** [SiLab-Bonn/testbeam_analysis#119]
After reloading a session and setting the pdf output under global options to on, no pdfs are created.

**Comment by leloup314** [comment]
Yep, that's true. Apparently plotting doesn't work at all, regardless whether you reload a session or not. Or does it for you?

**Comment by leloup314** [comment]
It appears that the [`plot_masked_pixels`](https://github.com/SiLab-Bonn/testbeam_analysis/blob/development/testbeam_analysis/tools/plot_utils.py#L47) works as expected when called in the script e.g. [here](https://github.com/SiLab-Bonn/testbeam_analysis/blob/development/testbeam_analysis/examples/eutelescope.py#L77) as well as when it's called from the GUI in order to plot onto the GUI. When it's called from the GUI though, with `plot=True` no PDF-files are created since they are all empty and empty PDFs are deleted by PDFPages by choice of `keep_empty=False`. So essentially everything works, except for that the PDFs in this case [here](https://github.com/SiLab-Bonn/testbeam_analysis/blob/development/testbeam_analysis/tools/plot_utils.py#L104) and [here](https://github.com/SiLab-Bonn/testbeam_analysis/blob/development/testbeam_analysis/tools/plot_utils.py#L119) are empty which is what I do not understand. If anyone has an idea, let me know.
